shopinvader_api_quotation/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Shopinvader Api Quotation",
+    "summary": "Shopinvader Quotation",
+    "version": "16.0.1.0.1",
+    "category": "e-commerce",
+    "website": "https://github.com/shopinvader/odoo-shopinvader",
+    "author": "Akretion",
+    "license": "AGPL-3",
+    "depends": ["crm", "fastapi", "shopinvader_schema_sale", "shopinvader_api_cart"],
```

Comment:
this module should depend on shopinvader_api_security_sale and  shopinvader_schema_sale (crm, shopinvader_api_cart can be removed)